locales/en/messages.jsonc

```diff
@@ -131,7 +131,7 @@
         "message": "Blocked Keywords"
     },
     "settings_blockedexperiences_keywords_description": {
-        "message": "Keywords can be added and removed to block experiences whose name matches one of the added keywords. Experiences that match will not display on some common pages."
+        "message": "Keywords can be added and removed to block experiences whose name matches one of the added keywords. These experiences will not be display on some common pages."
```

Comment:
"will not be display"?

Reply:
I'll fix that up

Reply:
Should be fixed in [53efd1e](https://github.com/RoSeal-Extension/I18n/pull/1/commits/53efd1edc2f1814c957c149d86eb1ab7019d6d87)